src/js/stores/__tests__/MesosStateStore-test.js

```diff
@@ -136,10 +247,81 @@ describe("MesosStateStore", function() {
       expect(result.fakeProp).toEqual("fake");
     });
 
-    it("returns null if node not found", function() {
+    it("returns undefined if node not found", function() {
       var result = MesosStateStore.getNodeFromID("nonExistentNode");
       expect(result).toEqual(undefined);
     });
+
+    it("returns null with invalid state", function() {
```

Comment:
should be named "returns undefined with invalid state"

Reply:
@ahoskins `returns null with invalid state` is correct. Take a look at the `MesosStateStore.getNodeFromID` and you will see that returns `null` not `undefined`.

I believe the test case here is incorrect having `toEqual(undefined)` since we already tested for `undefined` on line `250`.

I have fixed to return `null` instead